opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractJettyTest.java

```diff
@@ -84,11 +84,8 @@ protected void initTracing(ServletContextHandler context) {
         context.addFilter(new FilterHolder(new SpanFinishingFilter(mockTracer)), "/*",
             EnumSet.of(
                 DispatcherType.REQUEST,
-                DispatcherType.FORWARD,
```

Comment:
Why are the others being removed?

Reply:
It seems they are not needed - tests pass. 

I would rather remove them and add per case when we find out they are necessary.